assets/js/blocks/checkout/inner-blocks/checkout-actions-block/edit.tsx

```diff
@@ -83,12 +95,26 @@ export const Edit = ( {
 						/>
 					) }
 			</InspectorControls>
-			<Noninteractive>
-				<Block
-					showReturnToCart={ showReturnToCart }
-					cartPageId={ cartPageId }
-				/>
-			</Noninteractive>
+			<div className="wc-block-checkout__actions">
+				{ showReturnToCart && (
+					<ReturnToCartButton
+						link={ getSetting( 'page-' + cartPageId, false ) }
```

Comment:
I'm not sure why we need this in the editor? clicking it would take people outside the page, can this be an empty link in the editor?

Reply:
Removing `<ReturnToCartButton>` from the editor will lead to the fact that this link is no longer visible in the editor, given the changes that were necessary to make the `Place Order` button label editable:

<img width="557" alt="Screenshot 2022-12-06 at 09 20 35" src="https://user-images.githubusercontent.com/3323310/205792777-2ea40dac-ea12-41f6-8edb-ebba9fa7bba6.png">

However, you raised a good point and that link should indeed not be clickable in the editor. I wrapped it inside `<Noninteractive>` to prevent that a merchant can click the link.